apps/appointments/src/pages/admin/appointmentUnavailability/styles/index.module.scss

```
@@ -0,0 +1,34 @@
+  justify-content: space-between;
+  align-items: center;
+  background-color: $layer-01;
+  height: 54px;
+   position: sticky;
+  top: 0; /* Sticks to the very top */
+  z-index: 100; 
+}
+
+.title {
+  @include type-style("heading-compact-02");
+  color: $text-primary;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)